app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardActivity.kt

```diff
@@ -102,7 +102,7 @@ class DashboardActivity : DashboardElementActivity(), OnHomeItemClickListener, N
                 if (name?.trim { it <= ' ' }?.isEmpty() == true) {
                     name = profileDbHandler.userModel?.name
                 }
-                activityDashboardBinding.appBarBell.appTitleName.text = "$name's Planet"
+                activityDashboardBinding.appBarBell.appTitleName.text = name + getString(R.string.planet)
```

Comment:
1. this changes how the text appears 
current on master `Gideon's planet`
on this branch `Gideon planet`

2. warning is still not resolved **Follow this link on how to resolve it** [solution link](https://chatgpt.com/share/0214b7ce-f250-4e48-a1dd-44c1bcc8dd52)
<img width="772" alt="Screenshot 2024-07-01 at 20 43 06" src="https://github.com/open-learning-exchange/myplanet/assets/64019708/bab65fd7-857a-4d82-bb37-fa83bfbf3e85">

3. default and lite don't have the same changes
4. we have some conflicts